extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -1111,7 +1111,8 @@ export class ProjectsController {
 			const newProjFilePath = await this.createNewProject({
 				newProjName: projectInfo.projectName,
 				folderUri: vscode.Uri.file(projectInfo.outputFolder),
-				projectTypeId: constants.emptySqlDatabaseProjectTypeId
+				projectTypeId: constants.emptySqlDatabaseProjectTypeId,
+				sdkStyle: false
```

Comment:
I haven't tested the autorest flow with SDK-style projects, so I kept this as false for now to not change the current behavior